imap_processing/cli.py

```diff
@@ -162,11 +162,12 @@ def process(self):
         print(f"Processing IMAP-Ultra {self.level}")
 
 
-if __name__ == "__main__":
```

Comment:
You still need the `if __name__ == "__main__" call main() block for running it as `python cli.py` So it is best to leave that in there for either option.